### ISSUE TITLE
Add guidance about images of text

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#225]
In a website context the teaser images on [Colour > Social](https://govuk-brand-guidelines.netlify.app/colour/social/) would fail WCAG 1.4.5 Images of Text. In the context of examples in guidance they are okay and pass.

It would be good to add guidance to this page that mentions that in order to pass WCAG those types of images would need to have equivalent text beside the image.

I assume such a change wouldn't be part of the MVP but rather advice we should feed back to the Brand team.

Maybe we should use this to make the Brand team aware of a range of things to add to the guidelines. (TBC)

## Who needs to be involved

Accessibility specialist, content designer

## Who needs to review

Brand team
